css/main.css

```diff
@@ -0,0 +1,603 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+.mobile-nav ul {
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  width: 58%;
+}
```

Comment:
- [ ] As suggested by the previous reviewer, please align the links for the mobile version to the left as it is in the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)design. sticking to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design gives recruiters confidence in your ability to replicate exact spec.
- [ ] Also, the horizontal lines below the mobile nav links almost cover the entire width of the screen in the Figma design. Please try to fix this issue in your project. Here is how your project currently looks like: 
![origo](https://user-images.githubusercontent.com/73607512/184243604-ddc2da12-50d3-4328-b900-1138fb3fbf6d.PNG)